### PR TITLE
Fix assorted errors reported by the maven-qstools-plugin

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: ejb-remote/pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.jboss.quickstarts.eap</groupId>
-    <artifactId>jboss-ejb-remote-parent</artifactId>
```
Comment (rafabene) [inline ejb-remote/pom.xml:23]: Is this change from jboss-ejb-remote-parent to jboss-ejb-remote really expected ?

**Comment by rafabene** [comment]
Cool! You catch a lot of things

File: ejb-remote/pom.xml (hunk 1 above)
Comment (sgilda) [inline ejb-remote/pom.xml:23]: @rafabene : qstools reported this as a problem because the artifactId doesn't match the folder name.

**Comment by sgilda** [comment]
@rafabene : maven-qstools-plugin did it!

**Pull request merged**